code/game/objects/items/radio/headset.dm

```diff
@@ -161,7 +161,10 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	var/mob/living/carbon/human/wearer = null
 	var/headset_hud_on = FALSE
 	var/sl_direction = FALSE
-
+	/// Which hud this headset gives access too
+	var/hud_type = DATA_HUD_SQUAD
```

Comment:
consider renaming this to something relating to terragov to make it less confusing

Reply:
the define i mean